### PR TITLE
[SYCL][CI] Add sycl-ls log to Windows E2E testing logs

### PR DESCRIPTION
**Pull request by dm-vodopyanov**

### CODE REVIEW

File: devops/actions/e2e-tests-win/action.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ runs:
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
       cmake -GNinja -B build-e2e -S.\llvm\sycl\test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.targets }}" -DCMAKE_CXX_COMPILER="clang++" -DLEVEL_ZERO_LIBS_DIR="D:\github\level-zero_win-sdk\lib" -DLLVM_LIT="..\llvm\llvm\utils\lit\lit.py" ${{ inputs.cmake_args }}
+      sycl-ls --verbose
```
Comment (aelovikov-intel) [inline devops/actions/e2e-tests-win/action.yml:38]: Can we do

```
      echo "::group::sycl-ls --verbose"
      sycl-ls --verbose
      echo "::endgroup::
```

here? Not sure if that works on Windows, though.

**Review by aelovikov-intel — APPROVED** [review]

File: devops/actions/e2e-tests-win/action.yml (hunk 1 above)
Comment (dm-vodopyanov) [reply devops/actions/e2e-tests-win/action.yml:38]: Yeah, also not sure, will try

File: devops/actions/e2e-tests-win/action.yml [hunk 2]
```diff
@@ -32,6 +32,9 @@ runs:
   - name: Configure
     shell: cmd
     run: |
+      echo "::group::sycl-ls --verbose"
+      sycl-ls --verbose
+      echo "::endgroup::
       mkdir build-e2e
       set PATH=%GITHUB_WORKSPACE%\install\bin;%PATH%
```
Comment (aelovikov-intel) [inline devops/actions/e2e-tests-win/action.yml:39]: Don't think it can be earlier than `%PATH%` setting...

Comment (dm-vodopyanov) [reply devops/actions/e2e-tests-win/action.yml:39]: Oh, yes. Didn't noticed that...

**Pull request merged**